src/Pages/Login/Signup.scss

```
@@ -0,0 +1,104 @@
+@import 'src/Styles/variables.scss';
+.signupWrap {
```

```suggestion
@import 'src/Styles/variables.scss';

.signupWrap {
```
구분되는 요소들끼리는 한줄씩 띄워 구분해주세요!